From a7aed3c52bd2d5e3cddb80bfab7c0313bcf47242 Mon Sep 17 00:00:00 2001
From: Cameron Watters <cameron.watters@joyent.com>
Date: Fri, 4 Nov 2016 14:04:17 -0700
Subject: [PATCH] DOCKER-967 properly order extraHosts entries in /etc/hosts
 Reviewed by: Richard Kiene <richard.kiene@joyent.com> Reviewed by: Josh
 Wilsdon <josh@wilsdon.ca> Approved by: Josh Wilsdon <josh@wilsdon.ca>

---
 test/integration/cli-run-add-host.test.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/integration/cli-run-add-host.test.js b/test/integration/cli-run-add-host.test.js
index 2aff700..6096d75 100644
--- a/test/integration/cli-run-add-host.test.js
+++ b/test/integration/cli-run-add-host.test.js
@@ -66,7 +66,7 @@ test('docker run --add-host foo:1.2.3.4', function (tt) {
         var cmd = 'exec ' + containerName + ' grep foo /etc/hosts';
         cli.docker(cmd, function (err, stdout, stderr) {
             t.ifErr(err, 'docker CONTAINER grep foo /etc/hosts');
-            t.equal(stdout, 'foo\t1.2.3.4\n');
+            t.equal(stdout, '1.2.3.4\tfoo\n');
             t.end();
         });
     });
-- 
2.21.0

